src/vtt-segment-loader.js

```diff
@@ -32,6 +32,8 @@ export default class VTTSegmentLoader extends SegmentLoader {
     this.mediaSource_ = null;
 
     this.subtitlesTrack_ = null;
+
+    this.loaderType_ = 'subtitle';
```

Comment:
Not entirely sure if we want this in this pull request, but it's a bit weird for the `subtitleSegmentLoader_` to list itself as `main`. I had a hell of a time before I realized that `vtt-segment-loader` has nothing to do with 608 captions and all of that is done in the `mainSegmentLoader`